src/main/docs/guide/management/providedEndpoints/environmentEndpoint.adoc

```diff
@@ -1,20 +1,66 @@
 The environment endpoint returns information about the link:{api}/io/micronaut/context/env/Environment.html[Environment] and its link:{api}/io/micronaut/context/env/PropertySource.html[PropertySource]s.
 
-NOTE: Properties that may contain sensitive data are masked.
-
 == Configuration
 
-To configure the environment endpoint, supply configuration through `endpoints.env`.
+To enable and configure the environment endpoint, supply configuration through `endpoints.env`.
 
 .Environment Endpoint Configuration Example
 [source,yaml]
 ----
 endpoints:
   env:
-    enabled: Boolean    # default: true
+    enabled: Boolean    # default: false
     sensitive: Boolean  # default: true
 ----
 
+By default the endpoint will mask all values.
+To customize this masking you need to supply a Bean that implements link:{api}/io/micronaut/management/endpoint/env/EnvironmentEndpointFilter.html[EnvironmentEndpointFilter].
```

Comment:
You can use `api:management.endpoint.env.EnvironmentEndpointFilter[]`

Reply:
👍